src/js/base/IconLink.tsx

```diff
@@ -11,7 +11,7 @@ type IconLinkProps = {
 };
 
 export const IconLink = ({ color, name, replace, tip, to }: IconLinkProps) => (
-    <Link replace={replace} to={to}>
+    <Link replace={replace} to={to} aria-label={name}>
```

Comment:
Careful what you pass to aria-label: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-label